src/hooks/useSelectRevision.ts

```diff
@@ -10,8 +10,6 @@ const useSelectRevision = () => {
   const dispatch = useAppDispatch();
   const { enqueueSnackbar } = useSnackbar();
 
-  const searchResults = useAppSelector((state) => state.search.searchResults);
```

Comment:
searchResults was used to determine which revision to add to selectedRevisions, which I realize now was a bit of a convoluted way to accomplish this.

no longer needed since we are storing Revision objects directly in checkedRevisions state.